### PR TITLE
Improve CoronaTestStorage, only save non-generated value (EXPOSUREAPP-6007)

### PR DESCRIPTION
**Pull request by d4rken**
* Unit tests
* Only store values that are not generated out of other values.

### CODE REVIEW

**Review by BMItr — DISMISSED** [review]
lgtm.
Just ping if you will need reapproval soon.

**Review by BMItr — APPROVED** [review]

File: Corona-Warn-App/src/test/java/testhelpers/extensions/JsonExtensions.kt [hunk 1]
```diff
@@ -2,22 +2,29 @@ package testhelpers.extensions
...
+import com.google.gson.JsonArray
+import com.google.gson.JsonElement
 import com.google.gson.JsonObject
 import io.kotest.assertions.assertionCounter
 import io.kotest.assertions.collectOrThrow
 import io.kotest.assertions.eq.eq
 import io.kotest.assertions.errorCollector
 import okhttp3.mockwebserver.MockResponse
 
+private fun String.determineJsonType(): Class<out JsonElement> = when {
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/test/java/testhelpers/extensions/JsonExtensions.kt:14]: kinda hate this but i have no better idea 😆

**Review by kolyaopahle — APPROVED** [review]
LGTM ❤️

File: Corona-Warn-App/src/test/java/testhelpers/extensions/JsonExtensions.kt (hunk 1 above)
Comment (d4rken) [reply Corona-Warn-App/src/test/java/testhelpers/extensions/JsonExtensions.kt:14]: Test only :smirk:

**Pull request merged**